salesagility/SuiteCRM
# Hotfix sidebar suitep

## Description



When a user upgrades suitecrm, the modules which do not contain Menu.php will not display the icon in the sidebar correctly.

## Motivation and Context

This corrects the issue.

## How To Test This

* Remove Menu.php from a module
* Access the module
* You should see create and list action items with their respective icons.

## Types of changes


### Final checklist
